testsuite/tests/kuadrant/authorino/priority/test_sequence_api_key.py

```diff
@@ -0,0 +1,61 @@
+"""Test for api key identities, with different credential methods, sequential trigger according to their priorities"""
+import pytest
+
+from testsuite.utils import extract_from_response
+
+
+@pytest.fixture(scope="module")
+def second_label(module_label):
+    """Label for the second API key match"""
+    return f"{module_label}-second"
+
+
+@pytest.fixture(scope="module")
+def first_api_key(create_api_key, module_label):
+    """Create first API key"""
+    first_key = "first-key"
+    create_api_key("api-key-first", module_label, first_key)
+    return first_key
+
+
+@pytest.fixture(scope="module")
+def second_api_key(create_api_key, second_label):
+    """Create second API key"""
+    second_key = "second-key"
+    create_api_key("api-key-second", second_label, second_key)
+    return second_key
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, module_label, second_label):
+    """Add 2 API key identities with different credential method and priority to the AuthConfig"""
+    authorization.identity.api_key(
+        "priority-zero", match_label=module_label, credentials="authorization_header", priority=0
+    )
+    authorization.identity.api_key("priority-one", match_label=second_label, credentials="query", priority=1)
+
+    return authorization
+
+
+def test_priority_api_key(client, first_api_key, second_api_key, module_label, second_label):
+    """Send request with both credential methods at once and verify if high priority key was used."""
+    # verify that first API key is available to identify with and is used for identification
+    response = client.get("/get", headers={"authorization": "APIKEY " + first_api_key})
+    assert response.status_code == 200
+    label = extract_from_response(response)["identity"]["metadata"]["labels"]["group"]
```

Comment:
What do you mean exactly? `api_key` - the name of the identity, or identity name, `priority-zero` in this case?

In each case, I don't think it will be easier. The identity key of `AuthJson` returns the API key Kubernetes object in case of API key identity, so it is only possible to extract the assigned label, encoded API key, and name of the Kubernetes object.